package.json

```diff
@@ -6,8 +6,8 @@
   "scripts": {
     "coverage": "c8 --check-coverage node test/index.js",
     "test": "c8 node test/index.js",
-    "posttest": "eslint index.js",
-    "fix": "eslint index.js --fix"
+    "posttest": "eslint *.js",
```

Comment:
Why *.js and not just a dot?

Reply:
Because it flags a bunch of issues with unused variables in `README.md`, seems like a nuisance to apply the same linting rules to `js` code blocks, since you know they'll be partial code.

Reply:
I’m confused; how could eslint possibly warn on md files if `--ext=md` isn’t passed?

Either way, we should disable vis eslintignore and/or overrides, not by limiting what gets linted (dot catches nested files, for example, *.js doesn’t)

Reply:
> I’m confused; how could eslint possibly warn on md files if --ext=md isn’t passed?

Quick comment. Might be layered uses of eslint, like editor extension. I opened a PR prompted by md warnings in my environment! #35

Reply:
I just added README.md to ignores for now. I would have rather just added:

```
<!-- eslint-ignore no-unused-vars !-->
```

Tot he markdown file, which it seems should work based on `markdown/markdown` plugin information ... but none of the incantations I tried fixed it.